cms/djangoapps/contentstore/views/tests/test_videos.py

```diff
@@ -1278,6 +1278,7 @@ def test_video_transcript(self, preferences, is_video_transcript_enabled, error_
         response = json.loads(response.content.decode('utf-8')) if is_video_transcript_enabled else response
 
         self.assertEqual(status_code, expected_status_code)
+        error_message = error_message.replace("u'", "'") if six.PY3 else error_message
```

Comment:
The error message shouldn't have the unicode string indicator in it.   Where is that coming from?  We should probably fix it there to produce a unicode string object instead of a `repr` of a unicode string object.